recipes/util-linux/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "util-linux" %}
+{% set version = "2.34" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://mirrors.edge.kernel.org/pub/linux/utils/util-linux/v{{ version }}/util-linux-{{ version }}.tar.xz
+  sha256: 743f9d0c7252b6db246b659c1e1ce0bd45d8d4508b4dfa427bbb4a3e9b9f62b5
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  run_exports:
+    - {{ pin_subpackage('util-linux', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+  host:
+    - python
+    - zlib
+    - ncurses
+    - readline
+  run:
+    - python
+    - zlib
+    - ncurses
+    - readline
+
+test:
+  commands:
+    - test -f "${PREFIX}/lib/libmount.so"
+
+about:
+  home: https://en.wikipedia.org/wiki/Util-linux
+  license: GPL v2.0
```

https://github.com/karelzak/util-linux/blob/master/libmount/COPYING
```suggestion
  license: LGPL-2.0-or-later
```